2021-12-03 02:31:31,806 INFO    MainThread:53346 [wandb_setup.py:_flush():71] setting env: {}
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_init.py:_log_setup():371] Logging user logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_023131-1ce95v9b/logs/debug.log
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_023131-1ce95v9b/logs/debug-internal.log
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_init.py:init():404] calling init triggers
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'max_epochs': 150, 'batch_size': 100}
2021-12-03 02:31:31,807 INFO    MainThread:53346 [wandb_init.py:init():449] starting backend
2021-12-03 02:31:31,807 INFO    MainThread:53346 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-03 02:31:31,824 INFO    MainThread:53346 [backend.py:ensure_launched():199] starting backend process...
2021-12-03 02:31:31,835 INFO    MainThread:53346 [backend.py:ensure_launched():205] started backend process with pid: 53364
2021-12-03 02:31:31,843 INFO    MainThread:53346 [wandb_init.py:init():458] backend started and connected
2021-12-03 02:31:31,845 INFO    MainThread:53346 [wandb_init.py:init():519] updated telemetry
2021-12-03 02:31:31,860 INFO    MainThread:53346 [wandb_init.py:init():549] communicating current version
2021-12-03 02:31:33,133 INFO    MainThread:53346 [wandb_init.py:init():554] got version response 
2021-12-03 02:31:33,133 INFO    MainThread:53346 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-03 02:31:33,372 INFO    MainThread:53346 [wandb_init.py:init():592] starting run threads in backend
2021-12-03 02:31:36,085 INFO    MainThread:53346 [wandb_run.py:_console_start():1816] atexit reg
2021-12-03 02:31:36,086 INFO    MainThread:53346 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-03 02:31:36,086 INFO    MainThread:53346 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-03 02:31:36,088 INFO    MainThread:53346 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-03 02:31:36,088 INFO    MainThread:53346 [wandb_init.py:init():619] run started, returning control to user process
2021-12-03 02:31:36,088 INFO    MainThread:53346 [wandb_watch.py:watch():43] Watching
2021-12-03 02:31:36,094 INFO    MainThread:53346 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-03 02:31:36,094 INFO    MainThread:53346 [wandb_run.py:_restore():1758] restore
2021-12-03 02:31:38,615 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 02:31:38,852 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 02:31:39,391 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7709
}

2021-12-03 02:31:39,497 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7735
}

2021-12-03 02:31:39,600 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7735
}

2021-12-03 02:31:39,706 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:39,810 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:39,914 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:40,019 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:40,125 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:40,822 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}

2021-12-03 02:31:41,329 INFO    MainThread:53346 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7735
  total_bytes: 7735
}
local_info {
}

2021-12-03 02:31:42,626 INFO    MainThread:53346 [wandb_run.py:_append_files():2184] logging synced files
